templates/setenv.sh.erb

```diff
@@ -1,8 +1,7 @@
 #
 # One way to set the JIRA HOME path is here via this variable.  Simply uncomment it and set a valid path like /jira/home.  You can of course set it outside in the command terminal.  That will also work.
 #
-JIRA_HOME="<%= scope.lookupvar('jira::homedir') %>"
-
+JIRA_HOME="<%= @_data_dir %>"
```

Comment:
I believe this should remain as it was, otherwise it will break. Please see: [init.pp](https://github.com/voxpupuli/puppet-jira/blob/master/manifests/init.pp#L40) and [README](https://github.com/voxpupuli/puppet-jira#homedir)